commands/droplets_test.go

```diff
@@ -160,7 +160,9 @@ func TestDropletDelete(t *testing.T) {
 
 func TestDropletDeleteByTag(t *testing.T) {
 	withTestClient(t, func(config *CmdConfig, tm *tcMocks) {
-		tm.droplets.On("DeleteByTag", "my-tag").Return(nil)
```

Comment:
Since DeleteByTag now actually under the covers does a ListByTag, and then a delete on the returned list items by id, this means that the test needs to have mocks for ListByTag, and Delete instead of calling the droplet service DeleteByTag.

The reason that I changed the actual implementation to do a ListByTag and then do the deletes separately instead of the obvious answer of:
ListByTag
Show confirmation
DeleteByTag

Is to prevent the following from happening:
1. The user does a DeleteByTag named "test" and is shown a list of three droplets to delete.
2. something else, either the user in another process or script or something tags another droplet under the same tag, so there are now four droplets tagged with the tag "test"
3. The user confirms the DeleteByTag request. But instead of the original three droplets they were shown four are deleted which could lead to inadvertent data loss.

Because of the implementation, the mocks need to be enhanced.

Does that help answer your question?